petri/src/vm/hyperv/mod.rs

```diff
@@ -477,6 +478,7 @@ impl PetriVmmBackend for HyperVPetriBackend {
                 .context("failed to set ACL for igvm file")?;
 
             // TODO: only increase VTL2 memory on debug builds
+            // TODO: use vtl2_base_address_type
```

Comment:
What's the best way to model this? I'd rather `panic!`. That would require me to make the field in `OpenHclConfig` an `Option<...>`. That doesn't seem quite the right model, since the field isn't really optional. Carrying it around as an override doesn't fit in with the other fields here as well.